iowrappers/redis_client.go

```diff
@@ -182,74 +182,52 @@ func (r *RedisClient) getPlace(context context.Context, placeId string) (place P
 	return
 }
 
-// NearbySearchNotUsed is currently NOT used
-// to be used with the StorePlacesForLocation method
-// if no geocode in Redis, then we assume no nearby place exists either
-func (r *RedisClient) NearbySearchNotUsed(context context.Context, request *PlaceSearchRequest) ([]POI.Place, error) {
-	lat, lng, err := r.Geocode(context, &GeocodeQuery{
-		City:    request.Location.City,
-		Country: request.Location.Country,
-	})
-	if err != nil {
-		return nil, err
-	}
-	latLng := strings.Join([]string{fmt.Sprintf("%f", lat), fmt.Sprintf("%f", lng)}, ",")
-	sortedSetKey := strings.Join([]string{latLng, string(request.PlaceCat)}, "_")
-
-	placeIds, _ := r.client.ZRangeByScore(context, sortedSetKey, &redis.ZRangeBy{
-		Min: "0",
-		Max: fmt.Sprintf("%d", request.Radius),
-	}).Result()
-
-	res := make([]POI.Place, len(placeIds))
-
-	for idx, placeId := range placeIds {
-		res[idx], _ = r.getPlace(context, placeId)
-	}
-	return res, nil
-}
-
-func (r *RedisClient) NearbySearch(context context.Context, request *PlaceSearchRequest) (places []POI.Place, err error) {
-	requestCategory := strings.ToLower(string(request.PlaceCat))
+func (r *RedisClient) NearbySearch(ctx context.Context, req *PlaceSearchRequest) ([]POI.Place, error) {
+	requestCategory := strings.ToLower(string(req.PlaceCat))
 	redisKey := "placeIDs:" + requestCategory
 
-	requestLat, requestLng := request.Location.Latitude, request.Location.Longitude
+	requestLat, requestLng := req.Location.Latitude, req.Location.Longitude
 
-	searchRadius := request.Radius
+	searchRadius := req.Radius
 
 	if searchRadius > MaxSearchRadius {
```

Comment:
Curious where is MaxSearchRadius configured? is there a default value for this variable?